{
  "uid" : "4b4757e66a1912dae1a509f688f20b0f",
  "children" : [ {
    "name" : "Test defects",
    "children" : [ {
      "name" : "ValueError: {'code': -32000, 'message': 'connection already closed'}",
      "children" : [ {
        "name" : "test_rpc_call_eth_getBlockByHash",
        "uid" : "1dc157b3b100452d",
        "parentUid" : "cf7957bfdb0561efc74ada68d6693cf4",
        "status" : "broken",
        "time" : {
          "start" : 1649414919978,
          "stop" : 1649414921047,
          "duration" : 1069
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ ]
      }, {
        "name" : "test_rpc_call_eth_getBlockByNumber_via_numbers",
        "uid" : "4d3a23e7ba49cef3",
        "parentUid" : "cf7957bfdb0561efc74ada68d6693cf4",
        "status" : "broken",
        "time" : {
          "start" : 1649414923705,
          "stop" : 1649414924760,
          "duration" : 1055
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ ]
      }, {
        "name" : "test_rpc_call_eth_call",
        "uid" : "3f765b26e1071524",
        "parentUid" : "cf7957bfdb0561efc74ada68d6693cf4",
        "status" : "broken",
        "time" : {
          "start" : 1649414928630,
          "stop" : 1649414929692,
          "duration" : 1062
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ ]
      }, {
        "name" : "test_rpc_call_eth_getTransactionCount",
        "uid" : "be0002e7003c0942",
        "parentUid" : "cf7957bfdb0561efc74ada68d6693cf4",
        "status" : "broken",
        "time" : {
          "start" : 1649415064471,
          "stop" : 1649415065600,
          "duration" : 1129
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ ]
      }, {
        "name" : "test_rpc_call_eth_getTransactionByHash",
        "uid" : "f50852f15a5e6c52",
        "parentUid" : "cf7957bfdb0561efc74ada68d6693cf4",
        "status" : "broken",
        "time" : {
          "start" : 1649415070693,
          "stop" : 1649415071741,
          "duration" : 1048
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ ]
      }, {
        "name" : "test_too_low_gas_limit",
        "uid" : "f41dd12c9c5e45dd",
        "parentUid" : "cf7957bfdb0561efc74ada68d6693cf4",
        "status" : "broken",
        "time" : {
          "start" : 1649415100461,
          "stop" : 1649415101002,
          "duration" : 541
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ ]
      } ],
      "uid" : "cf7957bfdb0561efc74ada68d6693cf4"
    } ],
    "uid" : "bdbf199525818fae7a8651db9eafe741"
  }, {
    "name" : "Product defects",
    "children" : [ {
      "name" : "AssertionError: <AssertMessage.CONTAINS_ERROR: 'Contains the error data'>\nassert False\n +  where False = <bound method BasicTests.assert_no_error_object of <integration.tests.basic.test_rpc_calls.TestRpcCalls object at 0x7fcdbea7a130>>(JsonRpcErrorResponse(id=55, error={'code': -32000, 'message': 'connection already closed'}, jsonrpc='2.0'))\n +    where <bound method BasicTests.assert_no_error_object of <integration.tests.basic.test_rpc_calls.TestRpcCalls object at 0x7fcdbea7a130>> = <integration.tests.basic.test_rpc_calls.TestRpcCalls object at 0x7fcdbea7a130>.assert_no_error_object",
      "children" : [ {
        "name" : "test_rpc_call_eth_getLogs_via_tags[latest-latest]",
        "uid" : "24655cb340e4cfd0",
        "parentUid" : "74acd3fd7aec94cea11267db692a989a",
        "status" : "failed",
        "time" : {
          "start" : 1649414931428,
          "stop" : 1649414932265,
          "duration" : 837
        },
        "flaky" : true,
        "newFailed" : true,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "'latest'", "'latest'" ]
      } ],
      "uid" : "74acd3fd7aec94cea11267db692a989a"
    }, {
      "name" : "AssertionError: <AssertMessage.CONTAINS_ERROR: 'Contains the error data'>\nassert False\n +  where False = <bound method BasicTests.assert_no_error_object of <integration.tests.basic.test_rpc_calls.TestRpcCalls object at 0x7fcdbeb89d60>>(JsonRpcErrorResponse(id=16, error={'code': -32000, 'message': 'connection already closed'}, jsonrpc='2.0'))\n +    where <bound method BasicTests.assert_no_error_object of <integration.tests.basic.test_rpc_calls.TestRpcCalls object at 0x7fcdbeb89d60>> = <integration.tests.basic.test_rpc_calls.TestRpcCalls object at 0x7fcdbeb89d60>.assert_no_error_object",
      "children" : [ {
        "name" : "test_rpc_call_eth_getLogs_via_tags[earliest-latest]",
        "uid" : "daa77b5a79a70943",
        "parentUid" : "5b34828f1c52f62a3c4ce7ca738a9a53",
        "status" : "failed",
        "time" : {
          "start" : 1649414932276,
          "stop" : 1649414933114,
          "duration" : 838
        },
        "flaky" : true,
        "newFailed" : true,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "'earliest'", "'latest'" ]
      } ],
      "uid" : "5b34828f1c52f62a3c4ce7ca738a9a53"
    }, {
      "name" : "AssertionError: <AssertMessage.CONTAINS_ERROR: 'Contains the error data'>\nassert False\n +  where False = <bound method BasicTests.assert_no_error_object of <integration.tests.basic.test_rpc_calls.TestRpcCalls object at 0x7fcdbeb70280>>(JsonRpcErrorResponse(id=59, error={'code': -32000, 'message': 'connection already closed'}, jsonrpc='2.0'))\n +    where <bound method BasicTests.assert_no_error_object of <integration.tests.basic.test_rpc_calls.TestRpcCalls object at 0x7fcdbeb70280>> = <integration.tests.basic.test_rpc_calls.TestRpcCalls object at 0x7fcdbeb70280>.assert_no_error_object",
      "children" : [ {
        "name" : "test_rpc_call_eth_getLogs_via_tags[pending-latest]",
        "uid" : "58980b2d0537ea42",
        "parentUid" : "86c5bd303b9f72ba72a200301642a14f",
        "status" : "failed",
        "time" : {
          "start" : 1649414933124,
          "stop" : 1649414933960,
          "duration" : 836
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "'pending'", "'latest'" ]
      } ],
      "uid" : "86c5bd303b9f72ba72a200301642a14f"
    }, {
      "name" : "AssertionError: <AssertMessage.CONTAINS_ERROR: 'Contains the error data'>\nassert False\n +  where False = <bound method BasicTests.assert_no_error_object of <integration.tests.basic.test_rpc_calls.TestRpcCalls object at 0x7fcdbeb3aa90>>(JsonRpcErrorResponse(id=32, error={'code': -32000, 'message': 'connection already closed'}, jsonrpc='2.0'))\n +    where <bound method BasicTests.assert_no_error_object of <integration.tests.basic.test_rpc_calls.TestRpcCalls object at 0x7fcdbeb3aa90>> = <integration.tests.basic.test_rpc_calls.TestRpcCalls object at 0x7fcdbeb3aa90>.assert_no_error_object",
      "children" : [ {
        "name" : "test_rpc_call_eth_getLogs_via_tags[latest-earliest]",
        "uid" : "9c5af30d5dd8dcac",
        "parentUid" : "761461241487c871cfb781438dd5e275",
        "status" : "failed",
        "time" : {
          "start" : 1649414933971,
          "stop" : 1649414934808,
          "duration" : 837
        },
        "flaky" : true,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "'latest'", "'earliest'" ]
      } ],
      "uid" : "761461241487c871cfb781438dd5e275"
    }, {
      "name" : "AssertionError: <AssertMessage.CONTAINS_ERROR: 'Contains the error data'>\nassert False\n +  where False = <bound method BasicTests.assert_no_error_object of <integration.tests.basic.test_rpc_calls.TestRpcCalls object at 0x7fcdbea54640>>(JsonRpcErrorResponse(id=91, error={'code': -32000, 'message': 'connection already closed'}, jsonrpc='2.0'))\n +    where <bound method BasicTests.assert_no_error_object of <integration.tests.basic.test_rpc_calls.TestRpcCalls object at 0x7fcdbea54640>> = <integration.tests.basic.test_rpc_calls.TestRpcCalls object at 0x7fcdbea54640>.assert_no_error_object",
      "children" : [ {
        "name" : "test_rpc_call_eth_getCode",
        "uid" : "5ae6d1900b1bd133",
        "parentUid" : "07b20c804cb9a6f79df17d0d7e970db1",
        "status" : "failed",
        "time" : {
          "start" : 1649414937350,
          "stop" : 1649414938197,
          "duration" : 847
        },
        "flaky" : true,
        "newFailed" : true,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ ]
      } ],
      "uid" : "07b20c804cb9a6f79df17d0d7e970db1"
    } ],
    "uid" : "8fb3a91ba5aaf9de24cc8a92edc82b5d"
  } ],
  "name" : "categories"
}